sandboxes/with-perps/cannonfile.consumer.toml

```diff
@@ -0,0 +1,104 @@
+name = "perps-sandbox-consumer-example"
+version = "1"
+description = "Consumer of Perps Sandbox"
+
+#
+#
+# Provision Synthetix Sanbox with perps
+#
+#
+[import.perps_sanbox]
+source = "perps-sandbox"
+
+[setting.test_user]
+defaultValue = "0x70997970C51812dc3A010C7d01b50e0d17dc79C8" # PK 0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
+description = "Hardhat/Anvil second test account"
+
+[setting.test_user_account_id]
+defaultValue = "1"
+
+[invoke.test_user_creates_account]
+target = ["perps_sanbox.synthetix.CoreProxy"]
+from = "<%= settings.test_user %>"
+func = "createAccount(uint128)"
+args = ["<%= settings.test_user_account_id %>"]
+extra.test_user_account_id.event = "AccountCreated"
+extra.test_user_account_id.arg = 0
+
+[invoke.test_user_mints_10k_box]
+target = ["perps_sanbox.box_token.MintableToken"]
+from = "<%= settings.test_user %>"
+func = "mint"
+args = ["<%= parseEther('10000') %>", "<%= settings.test_user %>"]
+
+[invoke.test_user_approves_10k_box_for_CoreProxy_to_spend]
+target = ["perps_sanbox.box_token.MintableToken"]
+from = "<%= settings.test_user %>"
+func = "approve"
+args = [
+    "<%= imports.perps_sanbox.imports.synthetix.contracts.CoreProxy.address %>",
+    "<%= parseEther('10000') %>",
+]
+depends = ["invoke.test_user_mints_10k_box"]
+
+[invoke.test_user_deposits_1k_box]
+target = ["perps_sanbox.synthetix.CoreProxy"]
+from = "<%= settings.test_user %>"
+func = "deposit"
+args = [
+    "<%= extras.test_user_account_id %>",
+    "<%= imports.perps_sanbox.imports.box_token.contracts.MintableToken.address %>",
+    "<%= parseEther('1000') %>",
+]
+depends = ["invoke.test_user_approves_10k_box_for_CoreProxy_to_spend"]
+
+[invoke.test_user_delegates_900_box]
+target = ["perps_sanbox.synthetix.CoreProxy"]
+from = "<%= settings.test_user %>"
+func = "delegateCollateral"
+args = [
+    # examples of how to debug internal values line by line
+    "<%= /* console.log(extras) || */ extras.test_user_account_id %>",
```

Comment:
i actually ditnt know this was possible

Reply:
I wanted to output things into files but `require` was not available ;)